packages/w3/package.json

```diff
@@ -31,6 +31,7 @@
     "ora": "^5.4.1",
     "sade": "^1.7.4",
     "uint8arrays": "^3.0.0",
+    "w3name": "^1.0.0",
```

Comment:
Should we point to the latest 1.0.3 version?

Reply:
Good point. Done.